src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Listener.cs

```diff
@@ -77,7 +127,22 @@ private static void ConnectionCallback(UvStreamHandle stream, int status, Except
         /// </summary>
         /// <param name="listenSocket">Socket being used to listen on</param>
         /// <param name="status">Connection status</param>
-        protected abstract void OnConnection(UvStreamHandle listenSocket, int status);
+        private void OnConnection(UvStreamHandle listenSocket, int status)
+        {
+            UvStreamHandle acceptSocket = null;
+
+            try
+            {
+                acceptSocket = CreateAcceptSocket();
+                listenSocket.Accept(acceptSocket);
+                DispatchConnection(acceptSocket);
+            }
+            catch (UvException ex)
+            {
+                Log.LogError(0, ex, "Listener.OnConnection");
```

Comment:
Wouldn't this need to be the following then?

    Log.LogError(0, ex, $"{nameof(Listener)}.{nameof(Listener.OnConnection)}");

Not sure it's worth it since we never really expect the above to throw. If you search the Kestrel code base for "Log.LogError" you'll see a lot of places that need to change.